scripts/entrypoint.sh

```diff
@@ -14,7 +14,7 @@ if [[ "$1" == "mail" ]]; then
     MAIL_SMTP_ENABLE="TRUE"
     MAIL_DEBUG="TRUE"
 
-    if [[ -n "$2" ]]; then
```

Comment:
I don't like this change. Even though they do the same I prefer the `-n` for readability :)

Reply:
Yes, this change will reduce readability, I will re-add them.

Reply:
Thank you!